.github/workflows/build_and_test_tun2socks.yml

```diff
@@ -0,0 +1,103 @@
+name: Build and Test Tun2Socks
```

Comment:
Shall we just combine it with the client's build, so we don't need to add any new workflows? (after removing the Intra)

Reply:
+1

I recommend adding the build updates in the same PR you move the code:

- Having the code here, but using from the other repository is confusing.
- We need to make sure we can still build it, which we can only show by being able to build it
- We don't get any value from having code here we can't build